channels/ir.m3u

```diff
@@ -57,6 +61,8 @@ http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/ChannelIcons/gemusa-icon.png" group-title="Entertainment",GEM USA
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_usa/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Azerbaijani" tvg-logo="https://www.gunaz.tv/images/l1.png" group-title="News",Gunaz TV
+https://gtv.live-s.cdn.bitgravity.com/cdn-live/_definst_/gtv/live/multibitrate.smil/chunklist_w660946617_b450000.m3u8
```

Comment:
It is American channel (https://www.lyngsat.com/tvchannels/us/GunAz-TV.html)

Reply:
The same comment for AZ Star TV goes for this channel.